perception/traffic_light_recognition/traffic_light_classifier/README.md

```diff
@@ -1,29 +1,131 @@
+| `model_file_path` | str  | path to the model file               |
+| `label_file_path` | str  | path to the label file               |
+| `precision`       | str  | TensorRT precision, `fp16` or `int8` |
+| `input_c`         | str  | the channel size of an input image   |
+| `input_h`         | str  | the height of an input image         |
+| `input_w`         | str  | the width of an input image          |
+
+#### hsv_classifier
+
+| Name           | Type | Description                                    |
+| -------------- | ---- | ---------------------------------------------- |
+| `green_min_h`  | int  | the minimum hue of green color                 |
+| `green_min_s`  | int  | the minimum saturation of green color          |
+| `green_min_v`  | int  | the minimum value (brightness) of green color  |
+| `green_max_h`  | int  | the maximum hue of green color                 |
+| `green_max_s`  | int  | the maximum saturation of green color          |
+| `green_max_v`  | int  | the maximum value (brightness) of green color  |
+| `yellow_min_h` | int  | the minimum hue of yellow color                |
+| `yellow_min_s` | int  | the minimum saturation of yellow color         |
+| `yellow_min_v` | int  | the minimum value (brightness) of yellow color |
+| `yellow_max_h` | int  | the maximum hue of yellow color                |
+| `yellow_max_s` | int  | the maximum saturation of yellow color         |
+| `yellow_max_v` | int  | the maximum value (brightness) of yellow color |
+| `red_min_h`    | int  | the minimum hue of red color                   |
+| `red_min_s`    | int  | the minimum saturation of red color            |
+| `red_min_v`    | int  | the minimum value (brightness) of red color    |
+| `red_max_h`    | int  | the maximum hue of red color                   |
+| `red_max_s`    | int  | the maximum saturation of red color            |
+| `red_max_v`    | int  | the maximum value (brightness) of red color    |
+
+## Assumptions / Known limits
+
+<!-- Write assumptions and limitations of your implementation.
+
+Example:
+  This algorithm assumes obstacles are not moving, so if they rapidly move after the vehicle started to avoid them, it might collide with them.
+  Also, this algorithm doesn't care about blind spots. In general, since too close obstacles aren't visible due to the sensing performance limit, please take enough margin to obstacles.
+-->
+
+## (Optional) Error detection and handling
+
+<!-- Write how to detect errors and how to recover from them.
+
+Example:
+  This package can handle up to 20 obstacles. If more obstacles found, this node will give up and raise diagnostic errors.
+-->
+
+## (Optional) Performance characterization
+
+<!-- Write performance information like complexity. If it wouldn't be the bottleneck, not necessary.
+
+Example:
+
+  ### Complexity
+
+  This algorithm is O(N).
+
+  ### Processing time
+
+  ...
+-->
+
+## (Optional) References/External links
```

Comment:
```suggestion
## References/External links
```
Nit: This section has a content, so `(optional)` could be deleted.

Reply:
fixed at [4cf716b](https://github.com/tier4/AutowareArchitectureProposal.iv/pull/649/commits/4cf716bc9e935548eda95fa63648c5f5bdfdaa6d)